### PR TITLE
layer: dunfell release

### PR DESCRIPTION
**Pull request by dev-0x7C6**
Most changes imported from #26 and devel branch.

Tested with dunfell release.

### CODE REVIEW

**Comment by koenkooi** [comment]
Generic remark: please replace references of 'poky' with 'oe-core', since that's the actual upstream. No need to reference buggy downstreams.

**Comment by dev-0x7C6** [comment]
Can I do that without changing history? This commit was added long ago and this is already in meta-kodi:devel. Not sure if rebase will destroy my submodule references for me and others.

**Comment by koenkooi** [comment]
You'd have to amend the commit.

**Comment by dev-0x7C6** [comment]
> You'd have to amend the commit.

Is this advice or requirement ?

This commit was added in #14

According to https://linuxize.com/post/change-git-commit-message/#conclusion
I cannot amend this without changing history of `devel` branch, which may screw some users (count me as one).

Should I rebase 'koenkooi:devel' ?
Note that will:
- break history between 'koenkooi:devel' and "dev-0x7C6:master-next"
- after push-force to my "dev-0x7C6:master-next"
- this will break history between ''dev-0x7C6:master-next" and "dev-0x7C6:zeus"
- this will break history between ''dev-0x7C6:master-next" and "dev-0x7C6:dunfell"

this list also expand to my build enviroments forks and users of those...
and users of 'koenkooi:devel'

**Comment by dev-0x7C6** [comment]
@leon-anavi You can also voice your opinion

**Comment by leon-anavi** [comment]
@dev-0x7C6 I am still getting up to speed with the recent changes in this GitHub PR. On first thought I think you have done a great job! I second @koenkooi about `git commit --amend`. I often use it in similar cases.

As I mentioned before in another issue having this `devel` branch is a bit counter intuitive to me because I am used to having a branch for each Yocto/OE release, master and (eventually) master-next as in most of the other Yocto/OE layers. So in my opinion breaking the history isn't a big deal although it obviously is annoying for other developers.

File: recipes-multimedia/kodi-systemd-service/kodi-systemd-service/kodi-gbm.service [hunk 1]
```diff
@@ -1,8 +1,7 @@
 [Unit]
-Description=Kodi media thing
+Description=Kodi Media Center
 
 [Service]
-
 Environment=__GL_YIELD=USLEEP
 Environment=DISPLAY=:0.0
 Environment=WAYLAND_DISPLAY=wayland-0
```
Comment (avrahamshukron) [inline recipes-multimedia/kodi-systemd-service/kodi-systemd-service/kodi-gbm.service:7]: Should be removed for `gbm`

**Pull request closed without merging**